### PR TITLE
Add fb-scene-sample project

### PR DESCRIPTION
**Pull request by devloglogan**
Adds a sample project showcasing the FB extensions for scene and spatial anchors, and removes the same examples from the demo project.

I've also added a `.gdignore` file in the `samples/builds` directory, as David pointed out to me that exporting the file doesn't work unless that directory exists beforehand.

### CODE REVIEW

**Review by m4gr3d — CHANGES_REQUESTED** [review]

File: samples/fb-scene-sample/main.gd [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+]
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	xr_interface = XRServer.find_interface("OpenXR")
+	if xr_interface and xr_interface.initialize():
+		xr_interface.session_stopping.connect(self._on_session_stopping)
+		xr_interface.session_begun.connect(self.load_spatial_anchors_from_file)
+		var vp: Viewport = get_viewport()
+		vp.use_xr = true
```
[the remark this answers is not in the document]
Comment (devloglogan) [reply samples/fb-scene-sample/main.gd]: Even though this puts a couple things in the sample that end up unused, I think it makes sense to do this. In my latest push I've modified `main.gd` to follow the better start script template verbatim (comments and all). If we decide this is good, I'll update the rest of the samples accordingly.

[the remark this answers is not in the document]
Comment (m4gr3d) [reply samples/fb-scene-sample/main.gd]: > Edit: perhaps adding a comment at the top of the scripts indicating that the better start script doc was used as a starting template would be worthwhile?

Yeah, good idea!

**Review by m4gr3d — APPROVED** [review]
Looks good!

File: samples/fb-scene-sample/main.gd (hunk 1 above)
[the remark this answers is not in the document]
Comment (devloglogan) [reply samples/fb-scene-sample/main.gd]: I added a comment with a note / link to the doc, once the doc exists in stable I'll update it to that.

**Pull request merged**